src/index.js

```diff
@@ -29,116 +30,122 @@ const App = ()=> {
+  const minusOne = async (lineItem) => {
+    await api.minusOne({ lineItem, lineItems, setLineItems, cart });
   };
 
-  const decrement = async(lineItem) => {
-    await api.decrement({lineItem, lineItems, setLineItems, cart});
+  const decrement = async (lineItem) => {
+    await api.decrement({ lineItem, lineItems, setLineItems, cart });
   };
 
-  const cart = orders.find(order => order.is_cart) || {};
+  const cart = orders.find((order) => order.is_cart) || {};
 
-  const cartItems = lineItems.filter(lineItem => lineItem.order_id === cart.id);
+  const cartItems = lineItems.filter(
+    (lineItem) => lineItem.order_id === cart.id
+  );
 
-  const cartCount = cartItems.reduce((acc, item)=> {
-    return acc += item.quantity;
+  const cartCount = cartItems.reduce((acc, item) => {
+    return (acc += item.quantity);
   }, 0);
 
-  const login = async(credentials)=> {
+  const login = async (credentials) => {
     await api.login({ credentials, setAuth });
-  }
+  };
 
-  const signUp = async(credentials) => {
-    const response = await api.signUp({ credentials })
-    return response.data
-  }
+  const signUp = async (credentials) => {
+    const response = await api.signUp({ credentials });
+    return response.data;
+  };
 
-  const resetPassword = async(user, password) => {
-    const response = await api.resetPassword({ user, password })
-    return response.data
-  }
+  const resetPassword = async (user, password) => {
+    const response = await api.resetPassword({ user, password });
+    return response.data;
+  };
 
-  const resetUsername = async(user, username) => {
-    const response = await api.resetUsername({ user, username })
-    return response.data
-  }
+  const resetUsername = async (user, username) => {
+    const response = await api.resetUsername({ user, username });
+    return response.data;
+  };
 
-  const resetEmail = async(user, email) => {
-    const response = await api.resetEmail({ user, email })
-    return response.data
-  }
+  const resetEmail = async (user, email) => {
+    const response = await api.resetEmail({ user, email });
+    return response.data;
+  };
 
-  const logout = ()=> {
+  const logout = () => {
     api.logout(setAuth);
-    navigate('/account')
-  }
+    navigate("/account");
+  };
 
   const fetchUser = () => {
-    const user = users.data.find(user => user.username === auth.username)
-    return user
-  }
+    const user = users.data.find((user) => user.username === auth.username);
+    return user;
+  };
+  const fetchDetails = () => {
+    const details = products.data.find((product) => product.id === product_id);
+    return details;
+  };
 
   return (
     <div>
       {
         auth.id ? (
           <>
-            <div className='navi'>
             <nav>
-            <Link to='/home'>Home</Link>
-
-              <Link to='/products'>Courses ({ products.length })</Link>
+              <Link to='/products'>Products ({ products.length })</Link>
```

Comment:
this overrides home link and classname organization:

 <div className='navi'>
            <nav>
            <Link to='/home'>Home</Link>

              <Link to='/products'>Courses ({ products.length })</Link>